setup.cfg

```diff
@@ -37,8 +37,16 @@ tests_require =
     pytest-cov
     mock
 
+[options.entry_points]
+console_scripts =
+    gene_update = gene.cli:update_normalizer_db
+    gene_update_remote = gene.cli:update_from_remote
+    gene_dump = gene.cli:dump_database
```

Comment:
might rename these to `gene_norm_` rather than `gene_` just to be more specific

Reply:
I like that